scripts/job_uiuc.jobscript

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Usage:
+# 1. Copy to the directory where you have your files
+# 2. Update any needed environment variables and input file names in this script
+# 3. sbatch job_uiuc.jobscript
+
+#SBATCH --job-name="cardinal"
+#SBATCH --output="cardinal-%j.out"
+#SBATCH --partition=trial
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=16
+#SBATCH --mem=16G
+#SBATCH --account=RFST
+#SBATCH --export=ALL
+#SBATCH -t 03:00:00
+
+module purge
+module load cmake/3.18.4  
+module load python
+module unload gcc/7.2.0
+module load openmpi/4.1.1-gcc-9.2.0
+
+export CARDINAL_DIR=/projects/eng/shared/ajnovak2/cardinal
+export OPENMC_CROSS_SECTIONS=/projects/eng/shared/ajnovak2/cross_sections/endfb-vii.1-hdf5/cross_sections.xml
+export NEKRS_HOME=/projects/eng/shared/ajnovak2/cardinal/install
+
+#run cardinal
```

Comment:
I'd change this comment to `# run OpenMC case` because both of these are technically running cardinal.

Reply:
Done.